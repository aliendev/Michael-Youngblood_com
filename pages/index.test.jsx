import '@testing-library/jest-dom';
import { render, screen } from '@testing-library/react';
import Homepage from './index';


test('renders homepage unchanged', () => {
  const { container } = render(<Homepage />);
  expect(container).toMatchSnapshot();
});
