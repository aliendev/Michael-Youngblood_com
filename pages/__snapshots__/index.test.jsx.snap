// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <header
    class="p-3 bg-dark text-white"
  >
    <div
      class="container"
    >
      <div
        class="d-flex flex-wrap align-items-center justify-content-center justify-content-lg-start"
      >
        <a
          class="d-flex align-items-center mb-2 me-5 mb-lg-0 text-white text-decoration-none"
          href="/"
        >
          <svg
            class="bi bi-code-slash me-2"
            fill="currentColor"
            height="1em"
            viewBox="0 0 16 16"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <title>
              Michael Youngblood's Portfolio
            </title>
            <path
              d="M10.478 1.647a.5.5 0 1 0-.956-.294l-4 13a.5.5 0 0 0 .956.294zM4.854 4.146a.5.5 0 0 1 0 .708L1.707 8l3.147 3.146a.5.5 0 0 1-.708.708l-3.5-3.5a.5.5 0 0 1 0-.708l3.5-3.5a.5.5 0 0 1 .708 0m6.292 0a.5.5 0 0 0 0 .708L14.293 8l-3.147 3.146a.5.5 0 0 0 .708.708l3.5-3.5a.5.5 0 0 0 0-.708l-3.5-3.5a.5.5 0 0 0-.708 0"
            />
          </svg>
          Michael Youngblood
        </a>
        <ul
          class="nav col-12 col-lg-auto me-lg-auto mb-2 justify-content-center mb-md-0"
        >
          <li>
            <a
              class="nav-link px-2 text-white"
              href="/blog"
            >
              Blog
            </a>
          </li>
        </ul>
        <div
          class="text-end"
        >
          <a
            class="nav-link px-2 text-white"
            href="https://github.com/aliendev/Michael-Youngblood_com"
          >
            This Codebase
             
            <svg
              class="bi bi-github align-text-top ms-1"
              fill="currentColor"
              height="1em"
              viewBox="0 0 16 16"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <title>
                Code Repo for Michael-Youngblood.com
              </title>
              <path
                d="M8 0C3.58 0 0 3.58 0 8c0 3.54 2.29 6.53 5.47 7.59.4.07.55-.17.55-.38 0-.19-.01-.82-.01-1.49-2.01.37-2.53-.49-2.69-.94-.09-.23-.48-.94-.82-1.13-.28-.15-.68-.52-.01-.53.63-.01 1.08.58 1.23.82.72 1.21 1.87.87 2.33.66.07-.52.28-.87.51-1.07-1.78-.2-3.64-.89-3.64-3.95 0-.87.31-1.59.82-2.15-.08-.2-.36-1.02.08-2.12 0 0 .67-.21 2.2.82.64-.18 1.32-.27 2-.27s1.36.09 2 .27c1.53-1.04 2.2-.82 2.2-.82.44 1.1.16 1.92.08 2.12.51.56.82 1.27.82 2.15 0 3.07-1.87 3.75-3.65 3.95.29.25.54.73.54 1.48 0 1.07-.01 1.93-.01 2.2 0 .21.15.46.55.38A8.01 8.01 0 0 0 16 8c0-4.42-3.58-8-8-8"
              />
            </svg>
          </a>
        </div>
      </div>
    </div>
  </header>
  <div
    class="min-h-screen"
  >
    <main>
      <div
        class="container mx-auto px-5"
      >
        <div
          class="row flex-lg-row-reverse align-items-center g-5 py-5"
        >
          <div
            class="col-10 col-sm-8 col-lg-6"
          >
            <img
              alt="Michael Youngblood"
              class="d-block mx-lg-auto img-fluid border border-secondary p-2"
              height="500"
              loading="lazy"
              src="/images/profile-photo.jpg"
              width="700"
            />
          </div>
          <div
            class="col-lg-6"
          >
            <h1
              class="display-5 fw-bold lh-1 mb-3"
            >
              Michael Youngblood
            </h1>
            <p
              class="lead"
            >
              Software Engineer Manager (Web)
            </p>
            <p />
            <div
              class="d-grid gap-2 d-md-flex justify-content-md-start"
            />
          </div>
        </div>
      </div>
    </main>
  </div>
  <footer
    class="bg-accent-1 border-t border-accent-2"
  >
    <div
      class="container mx-auto px-5"
    >
      <footer
        class="d-flex flex-wrap justify-content-between align-items-center py-3 my-4 border-top"
      >
        <div
          class="d-flex align-items-center"
        >
          <span
            class="mb-4 mb-md-0 text-muted"
          >
            Copyright Â© 1982 - 
            2025
             
            <a
              href="/"
            >
              Michael Youngblood
            </a>
            . All rights reserved.
          </span>
        </div>
        <ul
          class="nav col-md-3 justify-content-end list-unstyled d-flex"
        >
          <li
            class="ms-3"
          >
            <a
              class="text-muted"
              href="https://aliendev.com"
            >
              AlienDev.com
            </a>
          </li>
          <li
            class="ms-3"
          >
            <a
              class="text-muted"
              href="https://github.com/aliendev/Michael-Youngblood_com"
            >
              <svg
                class="bi bi-github align-text-top"
                fill="currentColor"
                height="1em"
                viewBox="0 0 16 16"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <title>
                  Code Repo for Michael-Youngblood.com
                </title>
                <path
                  d="M8 0C3.58 0 0 3.58 0 8c0 3.54 2.29 6.53 5.47 7.59.4.07.55-.17.55-.38 0-.19-.01-.82-.01-1.49-2.01.37-2.53-.49-2.69-.94-.09-.23-.48-.94-.82-1.13-.28-.15-.68-.52-.01-.53.63-.01 1.08.58 1.23.82.72 1.21 1.87.87 2.33.66.07-.52.28-.87.51-1.07-1.78-.2-3.64-.89-3.64-3.95 0-.87.31-1.59.82-2.15-.08-.2-.36-1.02.08-2.12 0 0 .67-.21 2.2.82.64-.18 1.32-.27 2-.27s1.36.09 2 .27c1.53-1.04 2.2-.82 2.2-.82.44 1.1.16 1.92.08 2.12.51.56.82 1.27.82 2.15 0 3.07-1.87 3.75-3.65 3.95.29.25.54.73.54 1.48 0 1.07-.01 1.93-.01 2.2 0 .21.15.46.55.38A8.01 8.01 0 0 0 16 8c0-4.42-3.58-8-8-8"
                />
              </svg>
               
              Code Repo
            </a>
          </li>
        </ul>
        <ul
          class="nav col-md-2 justify-content-end list-unstyled d-flex"
        >
          <li
            class="ms-3"
          >
            <a
              class="text-muted"
              href="https://x.com/aliendev2"
            >
              <svg
                class="bi bi-twitter-x"
                fill="currentColor"
                height="1em"
                viewBox="0 0 16 16"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <title>
                  AlienDev on Twitter
                </title>
                <path
                  d="M12.6.75h2.454l-5.36 6.142L16 15.25h-4.937l-3.867-5.07-4.425 5.07H.316l5.733-6.57L0 .75h5.063l3.495 4.633L12.601.75Zm-.86 13.028h1.36L4.323 2.145H2.865z"
                />
              </svg>
            </a>
          </li>
          <li
            class="ms-3"
          >
            <a
              class="text-muted"
              href="https://www.twitch.tv/aliendev2"
            >
              <svg
                class="bi bi-twitch"
                fill="currentColor"
                height="1em"
                viewBox="0 0 16 16"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <title>
                  AlienDev on Twitch
                </title>
                <path
                  d="M3.857 0 1 2.857v10.286h3.429V16l2.857-2.857H9.57L14.714 8V0zm9.714 7.429-2.285 2.285H9l-2 2v-2H4.429V1.143h9.142z"
                />
                <path
                  d="M11.857 3.143h-1.143V6.57h1.143zm-3.143 0H7.571V6.57h1.143z"
                />
              </svg>
            </a>
          </li>
          <li
            class="ms-3"
          >
            <a
              class="text-muted"
              href="https://github.com/aliendev"
            >
              <svg
                class="bi bi-github"
                fill="currentColor"
                height="1em"
                viewBox="0 0 16 16"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <title>
                  Michael Youngblood on Github
                </title>
                <path
                  d="M8 0C3.58 0 0 3.58 0 8c0 3.54 2.29 6.53 5.47 7.59.4.07.55-.17.55-.38 0-.19-.01-.82-.01-1.49-2.01.37-2.53-.49-2.69-.94-.09-.23-.48-.94-.82-1.13-.28-.15-.68-.52-.01-.53.63-.01 1.08.58 1.23.82.72 1.21 1.87.87 2.33.66.07-.52.28-.87.51-1.07-1.78-.2-3.64-.89-3.64-3.95 0-.87.31-1.59.82-2.15-.08-.2-.36-1.02.08-2.12 0 0 .67-.21 2.2.82.64-.18 1.32-.27 2-.27s1.36.09 2 .27c1.53-1.04 2.2-.82 2.2-.82.44 1.1.16 1.92.08 2.12.51.56.82 1.27.82 2.15 0 3.07-1.87 3.75-3.65 3.95.29.25.54.73.54 1.48 0 1.07-.01 1.93-.01 2.2 0 .21.15.46.55.38A8.01 8.01 0 0 0 16 8c0-4.42-3.58-8-8-8"
                />
              </svg>
            </a>
          </li>
        </ul>
      </footer>
    </div>
  </footer>
</div>
`;
